src/utils/supabase.js

```diff
@@ -1,9 +1,9 @@
 import axios from 'axios';
 
 export const api_link =
-  process.env.NODE_ENV === 'development'
-    ? 'http://localhost:3001'
-    : 'https://dev-ophc7vkxsq-uc.a.run.app';
+  process.env.REACT_APP_ENV === 'dev'
+    ? 'https://dev-ophc7vkxsq-uc.a.run.app'
+    : 'https://api-ophc7vkxsq-uc.a.run.app';
```

Comment:
same thing

Reply:
Same comment as above

Reply:
this is related to the backend API right which should also be part of config